src/openrpc/certificationextension.json

```diff
@@ -0,0 +1,383 @@
+                    "params": [],
+                    "result": {
+                        "name": "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mP8/wcAAgMBApDPF90AAAAASUVORK5CYII=",
+                        "value": true
+                    }
+                },
+                {
+                    "name": "Screen capture and sends back the status",
+                    "params": [
+                        {
+                            "name": "uploadPath",
+                            "value": "http://example.com/upload"
+                        }
+                    ],
+                    "result": {
+                        "name": "success",
+                        "value": true
+                    }
+                }
+            ]
+        },
+        {
+            "name": "healthCheck",
+            "summary": "",
+            "tags": [
+                {
+                    "name": "capabilities",
+                    "x-provides": "xrn:firebolt:capability:certification:healthcheck",
+                    "x-allow-focus": true
+                }
+            ],
+            "params": [],
+            "result": {
+                "name": "healthCheckResult",
+                "summary": "Result of the interaction",
+                "schema": {
+                    "$ref": "#/components/schemas/HealthCheckResult"
+                }
+            },
+            "examples": [
+                {
+                    "name": "Snapshots for the device and process",
+                    "params": [],
+                    "result": {
+                        "name": "healthCheckResult",
+                        "value": {
+                            "load": 0.5,
+                            "deviceMemory": 0.5,
+                            "processMemory": 0.5,
+                            "cpu": 0.5,
+                            "connections": 10
+                        }
+                    }
+                }
+            ]
+        },
+        {
+            "name": "appPerformance",
+            "summary": "",
+            "tags": [
+                {
+                    "name": "capabilities",
+                    "x-provides": "xrn:firebolt:capability:certification:appperformancedata",
+                    "x-allow-focus": true
+                }
+            ],
+            "params": [
+                {
+                    "name": "appId",
+                    "required": true,
+                    "summary": "The id of the app for which the performance snapshot is required",
+                    "schema": {
+                        "type": "string"
+                    }
+                }
+            ],
+            "result": {
+                "name": "appPerformanceResult",
+                "summary": "Result of the interaction",
+                "schema": {
+                    "$ref": "#/components/schemas/AppPerformanceSnapshotResult"
```

Comment:
Wondering if we should align `healthCheck` and `appPerformance` methods, as they seem to be related and return similar values.

Maybe `systemHealthCheck` and `appHealthCheck`?